src/recipes.jl

```diff
@@ -0,0 +1,5 @@
+using RecipesBase
+
+## Plug into RecipesBase so plots of expressions treated like plots of functions
+@recipe f{T<:Basic}(::Type{T}, v::T) = lambdify(v)
+@recipe f{T<:Basic}(::Type{T}, v::T) = lambdify(v)
```

Comment:
Thanks for catching this. Fixed now.

On Fri, Jul 8, 2016 at 4:06 PM, Spencer Lyon <notifications@github.com>
wrote:

> In src/recipes.jl
> <https://github.com/symengine/SymEngine.jl/pull/27#discussion_r70133156>:
>
> > @@ -0,0 +1,5 @@
> > +using RecipesBase
> > +
> > +## Plug into RecipesBase so plots of expressions treated like plots of functions
> > +@recipe f{T<:Basic}(::Type{T}, v::T) = lambdify(v)
> > +@recipe f{T<:Basic}(::Type{T}, v::T) = lambdify(v)
>
> Looks like we have the same line of code twice?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/symengine/SymEngine.jl/pull/27/files/0d0dff7959b052c294d8cc63b2516054144f7f30#r70133156>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAZvTIiX6_tvLUQNKmx4mrPrI3an5nPzks5qTq3bgaJpZM4JINC3>
> .
>



-- 
John Verzani
Chair, Department of Mathematics
College of Staten Island, CUNY
verzani@math.csi.cuny.edu